### ISSUE TITLE
Rethink layout for root documentation page

### ISSUE THREAD

**Issue by samccone** [marionettejs/marionettejs.com#214]
Since our current doc page has a table of contents people seem to think the correct place to start is at the top (and I don't blame them)

![screen shot 2015-02-10 at 8 54 24 pm](https://cloud.githubusercontent.com/assets/883126/6140541/3ac21d3c-b167-11e4-9197-fe0424917063.png)

When the reality is the Application object is a pretty weak way to start off understanding how marionette works.

I propose we rethink how this page is done to give people a better idea of how to dive in.

Perhaps a three column system

```
| Views | Application | Helpers |
```

**Comment by peterblazejewicz** [comment]
I think docs needs its own embedded landing page, that quickly guides users between different docs topic. Angular has one:

https://github.com/angular/angular.js/blob/master/docs/content/guide/index.ngdoc
https://docs.angularjs.org/guide

In case of polymer one could read entire basic knowladge from the landing page embedded into context of docs (just like in angular):
https://github.com/Polymer/docs/blob/master/docs/polymer/polymer.md
https://www.polymer-project.org/docs/polymer/polymer.html

Marionette and Ember docs tend to be more *documentation* like without letting read something on the start (and people are lazy):
http://marionettejs.com/docs/current/
http://emberjs.com/api/
(but Ember has guides section).

I'm lazy and I also like to read something on the very first page:

http://docs.mongodb.org/manual/

**Comment by samccone** [comment]
http://insights.hotjar.com/static/heatmap/html/shared-heatmap.html?site=13560&heatmap=30965&token=fbbded4b6f5dd5f3a2b5e1237d15455e

**Comment by Spy-Seth** [comment]
I agree with the need to a specific landing page for the doc. A tree rows/colums layout with an introduction for each is a good start. 

The view section will be big, maybe it will need to be split in two part : basic view and advanced view.

**Comment by peterblazejewicz** [comment]
x-ref:
https://github.com/marionettejs/marionettejs.com/issues/75
